### DISCUSSION TITLE
Bug: Pull requests list shows inaccessible linked issues

### DISCUSSION THREAD

**Discussion by Marcono1234** [community/community#14971] [Pull Requests]
It looks like the pull requests list shows linked issues which are inaccessible to the current user.
An example for this can be seen [on `github/codeql`](https://github.com/github/codeql/pulls?q=is%3Apr+is%3Aclosed+%22Dataflow%3A+Cleanup+unused+column%22):
![Pull requests list](https://user-images.githubusercontent.com/11685886/163483768-43dcfb6d-5208-4d61-b501-4e49f706d3e4.png)

Note the linked issue icon on the right. However, clicking on it leads to a 404 page; most likely because the issue exists in a separate internal repository to which a regular user has no access.

Expected would be that those linked issues are not shown, because the user cannot access them anyway.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by Marcono1234** [reply]
Thanks for the link, I have reported it now! How exactly did you find the link to that form? If I remember correctly it was prominently shown on https://support.github.com/ a while ago, but that does not appear to be the case anymore.
